Vulnerability with tomcat-jasper-el

I’ve added `accesscontroltool-bundle (v3.0.2)` as a dependency in the project I’m working and since then, jenkins build fails due to some vulnerability related with `tomcat-jasper-el-8.5.66.jar `

Here's the jenkins error:
```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:5.2.4:check (default) on project contenthub-aem-core:
[ERROR]
[ERROR] One or more dependencies were identified with vulnerabilities:
[ERROR]
[ERROR] tomcat-jasper-el-8.5.66.jar: CVE-2021-33037, CVE-2020-8022
[ERROR]
[ERROR] See the dependency-check report for more details.
[ERROR]
```

--- a/accesscontroltool-bundle/pom.xml
+++ b/accesscontroltool-bundle/pom.xml
@@ -93,12 +93,12 @@
         <dependency>
             <groupId>org.apache.tomcat</groupId>
             <artifactId>tomcat-el-api</artifactId>
-            <version>8.5.66</version>
+            <version>8.5.70</version>
         </dependency>
         <dependency>
             <groupId>org.apache.tomcat</groupId>
             <artifactId>tomcat-jasper-el</artifactId>
-            <version>8.5.66</version>
+            <version>8.5.70</version>
         </dependency>
         <!-- due to https://bugs.openjdk.java.net/browse/JDK-8231581 OOTB JRE is not sufficient -->
         <dependency>

--- a/accesscontroltool-bundle/suppression.xml
+++ b/accesscontroltool-bundle/suppression.xml
@@ -2,7 +2,8 @@
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
   <suppress>
     <notes><![CDATA[
-   https://nvd.nist.gov/vuln/detail/CVE-2020-8022 marks it as vulnerable as it has version ranges for both tomcat 8 and 9.
+https://nvd.nist.gov/vuln/detail/CVE-2020-8022 marks it as vulnerable as it has version ranges without a lower bound for both tomcat 8 and 9.
+Reported at https://github.com/jeremylong/DependencyCheck/issues/3661.
    ]]></notes>
     <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat/tomcat\-jasper\-el@.*$</packageUrl>
     <cve>CVE-2020-8022</cve>

--- a/pom.xml
+++ b/pom.xml
@@ -341,7 +341,6 @@ Bundle-DocURL: https://github.com/Netcentric/accesscontroltool
                             <artifactId>aem-cloud-validator</artifactId>
                             <version>1.2.0</version>
                         </dependency>
-                        
                     </dependencies>
                 </plugin>
                 <plugin>
@@ -391,6 +390,19 @@ Bundle-DocURL: https://github.com/Netcentric/accesscontroltool
                         <tagNameFormat>@{project.version}</tagNameFormat>
                         <!-- skip rebuilding the release with GitHub actions (https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/) -->
                         <scmReleaseCommitComment>@{prefix} prepare release @{releaseLabel} [skip ci]</scmReleaseCommitComment>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.owasp</groupId>
+                    <artifactId>dependency-check-maven</artifactId>
+                    <version>6.3.1</version>
+                    <configuration>
+                        <cveValidForHours>24</cveValidForHours>
+                        <failBuildOnAnyVulnerability>true</failBuildOnAnyVulnerability>
+                        <skipProvidedScope>true</skipProvidedScope>
+                        <skipSystemScope>true</skipSystemScope>
+                        <retireJsAnalyzerEnabled>false</retireJsAnalyzerEnabled>
+                        <skip>true</skip><!-- only enable in specific submodules -->
                     </configuration>
                 </plugin>
             </plugins>
@@ -552,7 +564,6 @@ Bundle-DocURL: https://github.com/Netcentric/accesscontroltool
                     <plugin>
                         <groupId>org.owasp</groupId>
                         <artifactId>dependency-check-maven</artifactId>
-                        <version>6.2.0</version>
                         <executions>
                             <execution>
                                 <id>check-dependencies-for-vulnerabilities</id>
@@ -562,14 +573,6 @@ Bundle-DocURL: https://github.com/Netcentric/accesscontroltool
                                 </goals>
                             </execution>
                         </executions>
-                        <configuration>
-                            <cveValidForHours>24</cveValidForHours>
-                            <failBuildOnAnyVulnerability>true</failBuildOnAnyVulnerability>
-                            <skipProvidedScope>true</skipProvidedScope>
-                            <skipSystemScope>true</skipSystemScope>
-                            <retireJsAnalyzerEnabled>false</retireJsAnalyzerEnabled>
-                            <skip>true</skip><!-- only enable in specific submodules -->
-                        </configuration>
                     </plugin>
                 </plugins>
             </build>

